aip/general/0203.md

```diff
@@ -79,6 +81,15 @@ integers, or the unspecified value for enums) are indistinguishable from unset
 values, and therefore setting a required field to a falsy value yields an
 error. A corollary to this is that a required boolean must be set to `true`.
 
+### Optional
+
+The use of `OPTIONAL` indicates that a field is not required, nor covered by
```

Comment:
Remove the "nor covered by..." Only "required" is above it now.

Reply:
oops. That section should have been removed entirely. Thanks, fixed.